installer/pkg/workflow/destroy.go

```diff
@@ -30,7 +45,39 @@ func destroyTopologyStep(m *metadata) error {
 }
 
 func destroyJoinWorkersStep(m *metadata) error {
-	return runDestroyStep(m, joinWorkersStep)
+	waitTime := time.Duration(3)
```

Comment:
https://golang.org/pkg/time/#Duration
this would be 3 nanosecond?

Reply:
Fixed.